### DISCUSSION TITLE
on demand medication

### DISCUSSION THREAD

**Discussion by maulwurf87** [pewaru-333/HomeMedkit-App#75] [Q&A]
Hi,
is there a way to add an intake of medication on demand? I could not find anything outside a schedule. And I can only mark an intake of medication as taken when a pop-up appears, not bzw clicking on the schedule or anything. Is that expected (for now)?

Thanks for any replies.

**Comment by pewaru-333** [comment]
Hello. I don't clearly understand the first part. What does mean "on demand" in this case? For example, if you took one pill, you can edit left amount in this card, no reason to make one time intake. Secondly, you can also edit the status by clicking on intake item in "Taken" tab in "Intakes" menu.
